S2VX.Game/Story/Note/S2VXNote.cs

```diff
@@ -12,16 +11,15 @@ public abstract class S2VXNote : CompositeDrawable, IComparable<S2VXNote> {
 
         public Approach Approach { get; set; }
 
-        protected RelativeBox BoxOuter { get; } = new RelativeBox();
-        protected RelativeBox BoxInner { get; } = new RelativeBox();
+        public RelativeBox BoxOuter { get; } = new RelativeBox();
```

Comment:
Is there any reason why these need to be public?

Reply:
Notes.cs needs access to these now since Update isn't being called by the note itself, but by the container Notes class. If making it public is bad, I could possibly make a workaround with public methods in S2VXNote to edit these variables instead of having it directly accessible.

Reply:
I don't see Notes.cs using these, which is why I asked. Can you point me to where it's being used?

Reply:
Oh I forgot UpdatePlacement which accesses BoxOuter and BoxInner is in S2VXnote lol, I'll just make BoxOuter and BoxInner private